examples/lds01_dma/main.c

```diff
@@ -0,0 +1,191 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
+
+#include "event.h"
+#include "shell.h"
+
+#include "lds01.h"
+#include "lds01_params.h"
+
+/* This driver is not generic, it considers using our boards
```

Comment:
As said before I would add this comment inside the driver itself.

Reply:
I think I put it here because I wanted to have same header file for lds01 and lds01_dma, but since @ingroup is now different, I moved this comment to the header.